internal/web/hub.go

```diff
@@ -0,0 +1,45 @@
+package web
+
+// Hub maintains the set of active clients and broadcasts messages to the clients.
+type Hub struct {
```

Comment:
I think this name is a little confusing. I would rename it to `WsHub`.
(in that case, the file should also be renamed to `wshub.go`